cloud_governance/common/clouds/aws/price/price.py

```diff
@@ -50,3 +61,53 @@ def get_region_name(self, region_code):
             return data['partitions'][0]['regions'][region_code]['description']
         except IOError:
             return default_region
+
+    def get_ebs_cost(self, volume_type: str, region: str):
+        """
+        This method return the cost of ebs storage
+        @return:
+        """
+        volume_types = {
+            'gp': 'General Purpose',
+            'io': 'Provisioned IOPS',
+            'st1': 'Throughput Optimized HDD',
+            'standard': 'Magnetic',
+            'sc1': 'Cold HDD'
+        }
+        key = ''
+        for volume_key, value in volume_types.items():
+            if volume_key in volume_type:
+                key = value
+        return self.get_price(region=self.get_region_name(region), volume_type=key, aws_resource='ebs')
+
+    def get_resource_cost(self, resource: str, item_data: dict):
+        """
+        This method calculate ec2 cost from launch time or ebs per month in $
+        @return:cluster_cost_results
+        """
+        if resource == 'ec2' and item_data['State']['Name'] == 'running':
+            # Get current price for a given 'running' instance, region and os
+            ec2_type_cost = '0'
+            try:
+                ec2_type_cost = self.get_price(region=self.get_region_name('us-east-1'),
+                                               instance=item_data['InstanceType'], os='Linux')
+            except:
+                return 'NA'
+            ec2_lanuch_time = item_data['LaunchTime']
+            d1 = datetime.strptime(ec2_lanuch_time, "%Y-%m-%dT%H:%M:%S+00:00")
+            d2 = datetime.strptime(strftime("%Y-%m-%dT%H:%M:%S+00:00"), "%Y-%m-%dT%H:%M:%S+00:00")
+            diff = d2 - d1
+            diff_in_hours = diff.total_seconds() / 3600
+            ec2_cost = round(float(ec2_type_cost) * diff_in_hours, 3)
+            return round(ec2_cost, 3)
+        elif resource == 'ec2' and item_data['State']['Name'] != 'running':
+            return '0'
+        elif resource == 'ebs':
```

Comment:
Why did you keep EBS ?

Reply:
if I change here, It leads to making lot changes.